### PR TITLE
smalltalkci: add page

### PR DESCRIPTION
**Pull request by LinqLover**
/cc @fniephaus

### CODE REVIEW

File: pages/common/smalltalkci.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# smalltalkci
+
+> Framework for testing Smalltalk projects with GitHub Actions, Travis CI, AppVeyor, GitLab CI, and others.
+> More information: <https://github.com/hpi-swa/smalltalkCI#readme>
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:4]: Just use this
```suggestion
> More information: <https://github.com/hpi-swa/smalltalkCI>.
```

File: pages/common/smalltalkci.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# smalltalkci
+
+> Framework for testing Smalltalk projects with GitHub Actions, Travis CI, AppVeyor, GitLab CI, and others.
+> More information: <https://github.com/hpi-swa/smalltalkCI#readme>
+
+- Run tests for a config file:
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:6]: We don't use abbreviations
```suggestion
- Run tests for a configuration file:
```

File: pages/common/smalltalkci.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# smalltalkci
+
+> Framework for testing Smalltalk projects with GitHub Actions, Travis CI, AppVeyor, GitLab CI, and others.
+> More information: <https://github.com/hpi-swa/smalltalkCI#readme>
+
+- Run tests for a config file:
+
+`smalltalkci {{/path/to/.smalltalk.ston}}`
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:8]:
```suggestion
`smalltalkci {{path/to/.smalltalk.ston}}`
```

File: pages/common/smalltalkci.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run tests for a config file:
+
+`smalltalkci {{/path/to/.smalltalk.ston}}`
+
+- Run tests for the `.smalltalk.ston` configuration in the current directory:
+
+`smalltalkci`
+
+- Debug tests in headful mode (show VM window):
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:14]: `VM` is virtual machine, right?

File: pages/common/smalltalkci.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`smalltalkci --headful`
+
+- Specify prepared [S]malltalk image:
+
+`smalltalkci -s {{Squeak64-Trunk}}`
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:20]: We prefer long arguments if available
```suggestion
`smalltalkci --smalltalk {{Squeak64-Trunk}}`

```

File: pages/common/smalltalkci.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Specify custom Smalltalk image and VM:
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:22]:
```suggestion
- Specify a custom Smalltalk image and VM:
```

File: pages/common/smalltalkci.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+`smalltalkci --image {{/path/to/Smalltalk.image}} -- vm {{/path/to/vm}}`
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:24]:
```suggestion
`smalltalkci --image {{path/to/Smalltalk.image}} -- vm {{path/to/vm}}`
```

File: pages/common/smalltalkci.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+- Run tests for the `.smalltalk.ston` configuration in the current directory:
+
+`smalltalkci`
+
+- Debug tests in headful mode (show VM window):
+
+`smalltalkci --headful`
+
+- Specify prepared [S]malltalk image:
```
Comment (navarroaxel) [inline pages/common/smalltalkci.md:18]: I don't understand exactly what this example does.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/smalltalkci.md (hunk 4 above)
Comment (LinqLover) [reply pages/common/smalltalkci.md:14]: Correct. I think everyone who works with Smalltalk should have a basic understanding of the terms VM + image, so I would rather not explain this here in greater detail, would you agree with that?

File: pages/common/smalltalkci.md (hunk 8 above)
Comment (LinqLover) [reply pages/common/smalltalkci.md:18]: If `smalltalkci` is passed the `--smalltalk` option, it downloads and prepares a Smalltalk image for the given Smalltalk distribution and version. For instance, possible arguments would be:

```sh
smalltalkci --smalltalk Squeak64-Trunk  # bleeding edge Squeak image for 64-bit VMs
smalltalkci --smalltalk Squeak32-5.3  # Squeak version 5.3 image for 32-bit VMs
smalltalkci --smalltalk Pharo64-stable  # Pharo stable image for 64-bit VMs
```

This is described here in greater detail: <https://github.com/hpi-swa/smalltalkCI#images>

Hmm... Maybe change the description to this?

```suggestion
- Download and prepare a well-known [s]malltalk image for the tests:
```

Comment (navarroaxel) [reply pages/common/smalltalkci.md:18]: Yeah, sounds better, you can remove the `[]` because now you' re using the long argument format.

**Comment by navarroaxel** [comment]
@LinqLover please use UNIX line ending LF `\n` instead of windows CR LR `\r\n`.

File: pages/common/smalltalkci.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+`smalltalkci --smalltalk {{Squeak64-Trunk}}`
+
+
```
Comment (einverne) [inline pages/common/smalltalkci.md:22]: Here is an extra empty line.

**Review by einverne — CHANGES_REQUESTED** [review]
@LinqLover thanks for the new page. Please take a look at the extra line.

File: pages/common/smalltalkci.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+`smalltalkci`
+
+- Debug tests in headful mode (show VM window):
+
+`smalltalkci --headful`
+
+- Download and prepare a well-known smalltalk image for the tests:
+
+`smalltalkci --smalltalk {{Squeak64-Trunk}}`
+
```
Comment (LinqLover) [inline pages/common/smalltalkci.md:21]:
```suggestion
```

**Review by einverne — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by LinqLover** [comment]
Great, thank you!

**Pull request merged**